modules/ballerina-core/src/main/java/org/ballerinalang/bre/Context.java

```diff
@@ -50,6 +50,7 @@
     private BallerinaTransactionManager ballerinaTransactionManager;
     private DebugInfoHolder debugInfoHolder;
     private boolean debugEnabled = false;
+    private boolean nonBlockingActionCall = false;
```

Comment:
Please add this field in WorkerContext's populateContextPropsFromParent() method.